app/src/main/AndroidManifest.xml

```diff
@@ -39,8 +39,9 @@
             android:name="com.google.android.maps"
             android:required="true" />
 
+        <!--android:name="com.google.android.maps.v2.API_KEY"-->
         <meta-data
-            android:name="com.google.android.maps.v2.API_KEY"
+            android:name="com.google.android.geo.API_KEY"
```

Comment:
Doesn't seem to matter which one you use...and it errors if you use both. I think this one is a little more modern.